framework_tvb/tvb/interfaces/web/static/js/bursts.js

```diff
@@ -936,6 +936,54 @@ function displayBurstTree(selectedHref, selectedProjectID, baseURL) {
     $("#div-burst-tree").show();
 }
 
+function displayPseSimulationMessage(simulation_mesage){
```

Comment:
Have you searched for the previous JS code doing this count?
It should be either reused, or removed.

Reply:
I did now... I searched for 'pse' in js. files, but I couldn't find anything that looks like some kind of count of pse. Maybe it has been already deleted.

Reply:
Have you looked into range.js (or ranges.js)?

Reply:
See range.js after line 331.
Also, check all functions in that JS file for their usages, please.

Reply:
Yeah, I thought that those methods could be used in our case, but there is a problem. They use an id named "range_1" and "range_2", but the inputs for the ranges in the controller have "pse_param1_hi" and so on. So I strongly believe that those methods are used somewhere else, if used at all. I will check if they are used anywhere.

Reply:
A list of usages of all the functions in range.js:
- toggleRange - only used in number2component.html
- initRangeComponent - only used in the previous toggleRange
- _validateInput - used in the _updateRangeInterval function of this same file
- _updateRangeInterval - used only in initRangeComponent of this same file
- updateRangeValues - used only in bursts.js, function tryExpandRangers, line 374
- _getDataSelectRangeComponent - this seems not to be used anywhere, should I delete it?
- _getValues - used only in _displayRangeLabels of this same file
- _displayRangeLabels - used in multiple places in this same file: in updateRangeValues, initRangeComponent, slide, updateRangeInterval
- _prepareDataForValues - used in the same places as above 
- _getUnnormalizedValue - used two times in _displayRangeLabels, in _prepareDataForSubmit and _refreshFromToValues in this same file
- getNormalizedValue - used more times in updateRangeValues, in initRangeComponent and in updateRangeInterval in this same file
- _refreshFromToValues - used in initRangeComponent and slide in this same file
- displayRangeComponent - used in toggleRange of this same file and also used in stepsUpdate.js, line 29
- _getOpsForRanger - only used in RANGE_computeNrOfOps in this same file
- RANGE_computeNrOfOps - used in _prepareDataForSubmit and in disableRangeComponent of this same file, also used in stepsUpdate.js, line 68
-

Reply:
Ok, then this cleanup will be done in a separate task